OpenSim/Examples/SymbolicExpressionReporter/SymbolicExpressionReporter.cpp

```diff
@@ -260,9 +260,9 @@ begin( SimTK::State& s)
     // RESET STORAGE
     _resultStore.reset(s.getTime());
     // Populate list of names, initial values from s
-    Array<std::string> stateNames = _model->getStateVariableNames();
-    for(int i=0; i< stateNames.getSize(); i++){
-        _variables.insert(pair<string, double>(stateNames[i], 0.0));
+    std::vector<std::string> stateNames = _model->getStateVariableNames();
+    for(auto const& name : stateNames){
```

Comment:
The `const` applies to whatever is just to its left, with the exception that putting a const all the way to the left means it applies to the thing to its right. So `const auto&` and `auto const&` are the same.